test/sanity/validate-modules/ignore.txt

```diff
@@ -900,7 +900,8 @@ lib/ansible/modules/network/cnos/cnos_backup.py E322
 lib/ansible/modules/network/cnos/cnos_backup.py E323
 lib/ansible/modules/network/cnos/cnos_backup.py E326
 lib/ansible/modules/network/cnos/cnos_bgp.py E326
-lib/ansible/modules/network/cnos/cnos_command.py E326
+lib/ansible/modules/network/cnos/cnos_command.py E323
+lib/ansible/modules/network/cnos/cnos_config.py E323
```

Comment:
New ignores should not be added. The underlying issue(s) should be fixed. If those issues are in existing module_utils or docs fragments they should be fixed in a separate PR first.